WordPress/Classes/Models/AbstractPost.swift

```diff
@@ -2,6 +2,8 @@ import Foundation
 
 extension AbstractPost {
 
+    /// Returns the original post by navigating the entire list of revisions
+    /// until it reaches the head.
```

Comment:
Are there situations where the AbstractPost.original has a different original post?

Reply:
Yeah, if your create a revision R1 and then another one branching from it (R2), R2.original will point to R1 and not the original post.